src/test/java/se/diabol/jenkins/pipeline/domain/status/SimpleStatusTest.java

```diff
@@ -353,21 +353,21 @@ public void testCheckThatAllOnlyQueuedBuildIsResolvedAsQueued() throws Exception
         jenkins.buildAndAssertSuccess(project1);
         jenkins.waitUntilNoActivity();
 
-        List<Pipeline> pipelines = pipeline.createPipelineLatest(2, jenkins.getInstance());
+        List<Pipeline> pipelines = pipeline.createPipelineLatest(2, jenkins.getInstance(), false);
```

Comment:
extract false to a constant declared in the beginning of the class to declare the intent of the parameter, which then can be reused throughout the test file